File: apps/job-admin/src/app/core/types/models/Company.ts
import {BaseAuditableEntity} from './BaseAuditableEntity';

export interface Company extends BaseAuditableEntity {
  name?: string;
  description?: string | undefined;
  website?: string | undefined;
  email?: string;
  phone?: string | undefined;
  about?: string | undefined;
  eeo?: string | undefined;
  founded?: Date | undefined;
  size?: string | undefined;
  logo?: string | undefined;
  status?: string | undefined;
  isActive?: boolean;


}

File: services/micro-services/company-api/CompanyApi.Service/Infrastructure/Data/Entities/Company.cs
using Dapr.Client.Autogen.Grpc.v1;
using Elkhair.Dev.Common.Domain.Entities;

namespace CompanyApi.Infrastructure.Data.Entities;

public class Company : BaseAuditableEntity
{
    public required string Name { get; set; }
    public string? Description { get; set; }
    public string? Website { get; set; }
    public required string Email { get; set; }
    public string? Phone { get; set; }
    public string? About { get; set; }
    public string? EEO { get; set; }
    public DateTime? Founded { get; set; }
    public string? Size { get; set; }
    public string? Logo { get; set; }
    public required string Status { get; set; }
    
    public required int IndustryId { get; set; }
    public Industry Industry { get; set; } = null!;
}